scripts/research/lib/batch02_evidence_retention.py

```
@@ -0,0 +1,1036 @@
+                canonical_json_bytes(
+                    {
+                        "kind": "batch02_durable_retention_lock",
+                        "reservation_sha256": reservation.reservation_sha256,
+                        "evidence_ref": reservation.evidence_ref,
+                        "artifact_sha256": source_sha,
+                    }
+                )
+                + b"\n",
+            )
+        except Exception:
+            pass
```

Ruff (0.16.3)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(Exception): ...`

(SIM105)